wiki/Bot_account/de.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+  - bot accounts
+  - chat bot
+  - chat bots
+  - IRC bot
+  - IRC bots
+---
+
+# Bot-Account
+
+Ein **Bot-Account** ist eine spezielle Art eines osu!-Accounts, der von einem automatisierten Dienst anstelle einer realen Person betrieben wird. Auf der Webseite werden Bots mit dem weißen Gruppenabzeichen `BOT` gekennzeichnet und nur die `ich!`-Seite ihres Profils ist sichtbar. Bot-Accounts werden meistens dazu verwendet, "Chatbots" zu hosten, die über [Mehrspieler-Lobbies](/wiki/Client/Interface/Multiplayer) oder private Nachrichten verschiedene Funktionalitäten für die Community bereitstellen.
+
+Bot-Accounts können nur über den [Bewerbungsprozess](#einen-bot-account-erstellen) erstellt werden. Der Versuch, einen Bot-Account mit einer normalen Registrierung anzulegen, wird als Multi-Accounting betrachtet und ist gegen die [Community-Regeln](/wiki/Rules#community-regeln).
+
+## Vorteile von Bot-Accounts
+
+Der Hauptunterschied zwischen persönlichen und Bot-Accounts besteht in den Bandbreitenbeschränkungen, die bei der Versendung von [Chat](/wiki/Client/Interface/Chat_console)-Nachrichten gelten. Bot-Accounts haben höhere Grenzwerte, um zu ermöglichen, dass ihre Dienste mit mehr Nutzern interagieren können, ohne Gefahr zu laufen, dass der Account [stumm geschaltet](/wiki/Silence) wird:
+
+- Persönliche Accounts können jede 5 Sekunden 10 Nachrichten senden
+- Bot-Accounts können jede 60 Sekunden 300 Nachrichten senden
+
+Diese Beschränkungen gelten nur für private Nachrichten, `#multiplayer` und `#spectator`. Bot-Accounts dürfen in anderen Kanälen keine Nachrichten senden, auch keine unaufgeforderten, privaten Nachrichten.
+
+## Einen Bot-Account erstellen
+
+Wenn du überlegst, einen Chatbot einzurichten, beginne bitte damit, ihn auf deinem eigenen osu!-Account zu betreiben und stelle sicher, dass du die oben genannten Bandbreitenbeschränkungen für persönliche Accounts respektierst. Während dein Dienst wächst, kann es die Möglichkeit geben, dafür einen Bot-Account vom Account-Support-Team anzufragen.
+
+### Mindestanforderungen
+
+Bevor eine Anfrage geprüft wird, muss das Support-Team sicherstellen, dass der Bot folgende Kriterien erfüllt:
+
+- der Bot läuft seit mindestens 6 Monaten über deinen Account
+- der Bot ist vollständig Open-Source und hat eine vollumfängliche, öffentliche Dokumentation
+- der Bot wird jeden Monat von mindestens 50 einzelnen Nutzern verwendet
+- der Bot hält die Bandbreitenbeschränkungen für persönliche Accounts ein
+- der Bot sendet keine Nachrichten in öffentlichen Kanälen
+- der Bot sendet keine unerwünschten, privaten Nachrichten
```

Comment:
```suggestion
- der Bot sendet keine unerwünschten privaten Nachrichten
```

I'm not 100% sure on this, but I would argue that `private Nachricht` is a "Gesamtbegriff" as defined [here](https://www.duden.de/sprachwissen/sprachratgeber/Komma-zwischen-Adjektiven), and therefore the comma has to be removed.

Reply:
I think you are right. Trying to put an `und` in-between sounds weird.